src/rcl_bindings.cpp

```diff
@@ -83,6 +98,153 @@ NAN_METHOD(Init) {
   g_original_signal_handler = signal(SIGINT, Catch);
 }
 
+NAN_METHOD(GetParameterOverrides) {
+  RclHandle* context_handle = RclHandle::Unwrap<RclHandle>(
+      Nan::To<v8::Object>(info[0]).ToLocalChecked());
+  rcl_context_t* context =
+      reinterpret_cast<rcl_context_t*>(context_handle->ptr());
+
+  rcl_arguments_t *parsed_args = &(context->global_arguments);
+  rcl_params_t * params = NULL;
+  THROW_ERROR_IF_NOT_EQUAL(
+                  RCL_RET_OK,
+                  rcl_arguments_get_param_overrides(parsed_args, &params),
+                  rcl_get_error_string().str);
+
+  if (params == NULL) {
+    info.GetReturnValue().Set(Nan::Undefined());
+    return;
+  }
+
+  v8::Isolate * isolate = info.GetIsolate();
+  v8::Local<v8::Object> result = wrapParameters(params, isolate);
+
+  info.GetReturnValue().Set(result);
+}
+
+static const int PARAMETER_NOT_SET = 0;
+static const int PARAMETER_BOOL = 1;
+static const int PARAMETER_INTEGER = 2;
+static const int PARAMETER_DOUBLE = 3;
+static const int PARAMETER_STRING = 4;
+static const int PARAMETER_BYTE_ARRAY  = 5;
+static const int PARAMETER_BOOL_ARRAY = 6;
+static const int PARAMETER_INTEGER_ARRAY = 7;
+static const int PARAMETER_DOUBLE_ARRAY = 8;
+static const int PARAMETER_STRING_ARRAY = 9;
+
+/* 
+Convert parsed ros arguments to parameters.
+
+type Parameter = {
+  name: string,
+  type: number,
+  value: object
+}
+
+type Node = {
+  name: string,
+  parameters: array<Parameter>
+}
+
+parameters = array<Node>;
+*/
+static v8::Local<v8::Object> wrapParameters(rcl_params_t *parsed_args, v8::Isolate* isolate) {  // NOLINT(whitespace/line_length)
```

Comment:
I suggest we use `Nan` classes as much as possible, as the changes of v8 could bring incompatibility.

- [Nan::New()](https://github.com/nodejs/nan/blob/master/doc/new.md#api_nan_new) to create javascript objects.
- [Nan::Set()](https://github.com/nodejs/nan/blob/master/doc/maybe_types.md#api_nan_set) to set the elements of an array.

Reply:
done. Thanks for recommendation.